src/components/chatProfile.tsx

```diff
@@ -0,0 +1,54 @@
+import styled from 'styled-components';
+import { useNavigate } from "react-router-dom";
+
+const ChatProfile = ({img, name, message, id}) => {
+  const navigate = useNavigate();
+  const move = (id:number) => {
+    navigate('/chatRoom', {
+      state: {
+        roomId: id,
+      }
+    });
+  };
+  return (
+    <Container  onClick={() => {move(id)}}>
+      <ProfileImg src={`img/${img}.jpeg`}/>
+      <ColumnContainer>
+        <Name> {name} </Name>
+        <Message> {message} </Message>
+      </ColumnContainer>
+    </Container>
+  );
+};
+
+export default ChatProfile;
```

Comment:
```suggestion
export default React.memo(ChatProfile);
```
요렇게 하시면 리렌더링 방지할 수 있어요!
[참고 링크](https://ui.toast.com/weekly-pick/ko_20190731)